authorization/invitation/repository/invitation.go

```diff
@@ -192,6 +192,26 @@ func (m *GormInvitationRepository) Delete(ctx context.Context, id uuid.UUID) err
 
 	obj := Invitation{InvitationID: id}
 
+	var invitationRoles []InvitationRole
+
+	err := m.db.Where("invitation_id = ?", id).Find(&invitationRoles).Error
+	if err != nil && err != gorm.ErrRecordNotFound {
+		return errs.WithStack(err)
+	}
+
+	for _, r := range invitationRoles {
```

Comment:
Could you please add/update the repo test to validate deletion of invitationRoles too?

Reply:
Done